Install-Guide/Community_Packages.md

```diff
@@ -34,3 +34,11 @@ A **—** means we have no plans to build the version for the repository.
 |                   |Leap 42.X    |    X     |    X     |    X     |    —     |
 |                   |SLES11       |    —     |    —     |    —     |    X     |
 |                   |SLES12       |    X     |    X     |    X     |    X     |
+
+
+	SLES12 	X
```

Comment:
There is something wrong here. You could use any online markdown editor to check rendering.